text-ui-test/input.txt

```diff
@@ -0,0 +1,13 @@
+todo beat elite four
+todo find missingno
+deadline ride s.s. Anne /by tomorrow
+deadline evolve /by level 16
+event release of mystery dungeon dx /by yesterday-ish
+event year of the pikachu /by 25 Jan 2020
+list
+done 6
+done 4
+done 3
+list
+bye
+
```

Comment:
Nice test cases that explore possible commands. Maybe you can also include some more test cases that cover exceptions!